deno.json

```diff
@@ -1,6 +1,11 @@
 {
   "tasks": {
-    "dev": "deno run --watch --no-check --allow-read --allow-write --allow-run --allow-env --allow-net main.ts"
+    "dev": "deno run --watch --allow-read --allow-write --allow-run --allow-env --allow-net main.ts"
+  },
+  "compilerOptions": {
+    "types": [
+      "https://raw.githubusercontent.com/denoland/doc_components/cad26fe716a3775fd952aff6caacb191bc35c2b9/jsx.d.ts"
```

Comment:
What's this about?

Reply:
its necessary for doc_components/services.ts usage of the h/Fragment preact functions (else everything errors with `TS7026 [ERROR]: JSX element implicitly has type 'any' because no interface 'JSX.IntrinsicElements' exists.`)

Reply:
Can you leave a comment? Is this a doc_components specific problem?

Reply:
Well, its being pulled from doc_components, so it seems fairly obvious that it is needed for doc_components